src/config/config.ts

```diff
@@ -32,3 +41,34 @@ export const getHttpProviderUri = (networkId: string): string => {
   if (networkId === "homestead") return `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`;
   return `https://${networkId}.infura.io/v3/${INFURA_PROJECT_ID}`;
 };
+
+export const getEtherscanNetworkApiDetails = (chain: string, chainId: number): EtherscanNetworkApiDetails => {
```

Comment:
do we need to pass in `chain`? looks like they are set later depending on some fixed `chainId`